src/detect_diff.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+git add -A
+
+if git diff --cached --quiet; then
+	result=""
+else
+	result="差分あり"
```

Comment:
```suggestion
	has_diff="false"
else
	has_diff="true"
```

内部的にしか使わない感じなので、bool値っぽい感じにしても良さそう。

Reply:
......と思ったけど、今のやり方の方が変更漏れが少なくていいか。